### ISSUE TITLE
Link to API page is broken

### ISSUE THREAD

**Issue by mkevac** [EvaTheSalmon/coub-dl-liked#1]
The link from README (https://coub.com/api/v2/users/me) gives me 404. Any solutions?

<img width="917" alt="image" src="https://github.com/EvaTheSalmon/coub-dl-liked/assets/123544/24cf39b4-3f2c-45b9-8a85-b61ad40529c2">

**Comment by mrcsin** [comment]
Hello,

I haven't updated README in a while. Obtaining an API key via the following link is not working for the past years since coub.com got owned by a new company, I have some issues with API myself.

Since I've stored my API key locally for a long time, and because there hasn't been much activity on the project, I didn't really look into fixing it. Unfortunately coub has deprecated and removed it's own developer docs, see the [following](https://web.archive.org/web/20220315172303/https://coub.com/dev/docs/Coub+API%2FOverview). So far as I see, coub team is not interested in side project like this. But you may contact coub support to see what they say, I would be interested in their answer too. 

Suddenly other major [projects](https://github.com/topics/coub-api) use old api version, so no solution is present or I didn't find it.

TLDR: I have no workaround this issue

**Comment by mrcsin** [comment]
Ok, quick update here. I found the ```api_key``` in the page source: view-source:https://coub.com/hot 

![image](https://github.com/EvaTheSalmon/coub-dl-liked/assets/18466855/e6fea274-cbc8-4862-ae20-7f4669cc9dc6)

I'll update docs in a while
